app/src/main/java/com/example/iou/activities/SignupActivity.java

```diff
@@ -73,14 +82,14 @@ public void done(ParseUser user, ParseException e) {
                 if (e != null) {
                     return;
```

Comment:
Swallowing an exception is a code smell. We'd likely want to surface an error message up to the user that something has gone wrong.